backend/create_dummy_data.py

```diff
@@ -321,7 +332,7 @@
             "correct_option": "Margalit",
         },
     ],
-    "published": "True",
+    "published": True,
```

Comment:
squash :clap: and merge :twisted_rightwards_arrows:

Reply:
nice